sdsim/Assets/Scripts/Race/LapTimer.cs

```diff
@@ -82,16 +85,20 @@ public void OnCollideFinishLine()
         {
             float timeNow = GetCurrentMS();
             float lapTime = GetCurrentLapTime();
-            
-            lapTimes.Add(lapTime);
 
-            if (lapTime < bestTime)
+            if (lapTime > 50.0)
```

Comment:
Work-around for not correct working laptrigger on finish line in new 'long_road' track from Tawn. No laptrigger for below-second laps.